src/components/Card/__stories__/CardShowcase.tsx

```diff
@@ -0,0 +1,167 @@
+                        return (
+                            <div key={theme}>
+                                <ReactCopyToClipboard text={getContainerCard('filled', theme)}>
+                                    <div>
+                                        <Card
+                                            className="card-showcase-stories"
+                                            view="filled"
+                                            theme={theme}
+                                        >
+                                            {' '}
+                                        </Card>
+                                    </div>
+                                </ReactCopyToClipboard>
+                                <div className="card-showcase-stories__desc">theme {theme}</div>
+                            </div>
+                        );
+                    })}
+                </div>
+                <div className="panel-showcase-stories__head">Raised view</div>
+                <div className="panel-showcase-stories__cards">
+                    <div>
+                        <ReactCopyToClipboard text={getContainerRaisedCard()}>
+                            <div>
+                                <Card className="card-showcase-stories" view="raised">
+                                    {' '}
+                                </Card>
+                            </div>
+                        </ReactCopyToClipboard>
+                    </div>
+                </div>
+            </div>
+            {/* Action card */}
+            <div className="panel-showcase-stories">
+                <div className="panel-showcase-stories__title">Action</div>
+                <div className="panel-showcase-stories__desc">Action cards</div>
+                <div className="panel-showcase-stories__cards">
+                    <div>
+                        <ReactCopyToClipboard text={getActionCard()}>
+                            <div>
+                                <Card
+                                    className="card-showcase-stories"
+                                    type="action"
+                                    onClick={() => {}}
+                                >
+                                    {' '}
+                                </Card>
+                            </div>
+                        </ReactCopyToClipboard>
+                        <div className="card-showcase-stories__desc">type action</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function getSelectionCard(view: CardView) {
+    return (
+        `<Card\rtype="selection" view="${view}" onClick={() => {}}\r` +
+        `className="" selected={} disabled={}\r> </Card>`
+    );
+}
+
+function getContainerCard(view: CardView, theme: CardTheme) {
+    return `<Card\rtype="container" theme="${theme}" view="${view}" className=""\r> </Card>`;
```

Comment:
Single-quoted strings can be used instead.

Reply:
This should be done by linter :P

Reply:
I've refactored stories 6c24cf8